lib/scripts/create_keywords_yaml.rb

```diff
@@ -0,0 +1,18 @@
+INDENT = "  "
+
+File.open("keywords.yaml", "w") do |file|
+  tags = {}
+
+  GenericFile.all.each do |gf|
+    gf.tag.each do |tag|
+      slug = tag.parameterize
+
+      if tags[slug]
+        next
+      else
+        tags[slug] = tag
+        file.write("- id: #{slug}\n#{INDENT}title:\n#{INDENT*2}en: #{tag}\n")
```

Comment:
I think I like this simple approach because it creates the yaml file iteratively. I have been using a yaml library on the Python side to write the yaml, but it takes the whole data as input which is not great. The only thing I wonder here is if the content of `#{tag}` could interfere with the yaml syntax... e.g. if it has colons, commas, dashes, brackets... Do our keywords look like that?

Feel free to make a PR to https://github.com/galterlibrary/digitalhub-2 with the generated keywords.yaml when you have it.

Reply:
I was thinking we could wrap it in quotes. I think that is valid YAML syntax.